client/main.lua

```diff
@@ -202,11 +203,13 @@ local function carjackVehicle(target)
     lib.requestAnimDict('mp_am_hold_up')
     local vehicle = GetVehiclePedIsUsing(target)
     local occupants = getPedsInVehicle(vehicle)
-    for _, occupant in ipairs(occupants) do
+    for p = 1, #occupants do
+        local occupant = occupants[p]
```

Comment:
found something interesting related to this
![obraz](https://github.com/Qbox-project/qbx_vehiclekeys/assets/36608116/85ed9889-ae8d-4083-a3a5-98e4d42ac742)

```lua
benchmark()(100, {
    numeric = function()
        for i = 1, #data do
            x = data[i]
        end
    end,
    numericWithAassignment = function()
        for i = 1, #data do
            local d = data[i]
            x = d
        end
    end,
    next = function()
        for i, v in next, data do
            x = v
        end
    end,
    pairs = function()
        for i, v in pairs(data) do
            x = v
        end
    end,
    ipairs = function()
        for i, v in ipairs(data) do
            x = v
        end
    end,
    each = function()
        for i, v in each(data) do
            x = v
        end
    end,
})
```
it is repeatable

Reply:
There you see why I opt to use numeric instead of ipairs, also I find ipairs ugly

Reply:
In critical sections it is better to use numeric loops.
Ipairs is more readable, so in light operations it should be preferable.

Reply:
Not to me, I prefer the speed of numeric loops over readability and I stay away from ipairs entirely just because I don't like its appearance either.

Reply:
Brought this debate up to the team. I think best way forward is for the team to decide on a preference and make it consistent across the code base. I don't have a strong opinion either way.